packages/feedback/src/core/sendFeedback.ts

```diff
@@ -23,6 +29,9 @@ export const sendFeedback: SendFeedback = (
     throw new Error('No client setup, cannot send feedback.');
   }
 
+  if (options.tags) {
```

Comment:
```suggestion
  if (options.tags && Object.keys(options.tags).length) {
```

Ensure to simply not set this at all if no tags are configured, this is nicer IMHO? :)